src/components/Shared/About.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
```

Comment:
It should be under Settings folder, and rename the component to `SectionWrapper` instead.

Reply:
But this has nothing to do with Settings folder, it is for the overview, blogs, etc static pages